index.js

```diff
@@ -133,9 +133,9 @@ async function send (ctx, path, opts = {}) {
     }
     ctx.set('Cache-Control', directives.join(','))
   }
-  ctx.type = type(path)
   ctx.body = fs.createReadStream(path)
-
+  ctx.type = ctx.res.type ? ctx.res.type : type(path);
```

Comment:
we can put setHeaders before set body. i tested and it worked. But this code should be added instead of earlier for setting the content type.
`ctx.type = ctx.res.type ? ctx.res.type : type(path);`

If the type is not set, then it should set from file extensions.
Thanks